packages/@okta/courage-for-signin-widget/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "webpack"
   },
   "devDependencies": {
-    "@okta/courage": "4.6.0-beta.2670.g173eed7",
+    "@okta/courage": "^4.6.0-beta.2702.g116a94b",
```

Comment:
probably want to be exact version `4.6.0-beta.2702.g116a94b`?

Reply:
Ah, didn't see that `^`....  I just ran the instructions on the readme for it.  ill remove that

Reply:
interesting.. usually `yarn upgrade` will pin to specific version..
btw, the yarn.lock wasn't updated.